### PR TITLE
[docs] [external-assets] - Round 2

### PR DESCRIPTION
**Pull request by erinkcochran87**
## Summary & Motivation

This PR does another round of edits on the External Assets concept page.

### CODE REVIEW

File: docs/content/concepts/assets/external-assets.mdx [hunk 1]
```diff
@@ -124,7 +123,7 @@ height={1654}
 </TabItem>
 </TabGroup>
 
-### Fully-managed assets with external asset dependencies
+### Dagster-native assets with external asset dependencies
```
Comment (erinkcochran87) [inline docs/content/concepts/assets/external-assets.mdx:126]: Thoughts on this terminology?

**Comment by erinkcochran87** [comment]
These things came up during dogfooding that I could use some help answering:

- [ ] 149: Should `external_assets_from_specs` have the asterisk? 
- [ ] 313: Can `report_runless_asset_event` be used for non-external assets?
- [ ] 320: Any reason not to include the instance call (`DagsterInstance.get()`?

Also, should we include anything about Pipes in here? Ex: "If you want to trigger the execution of external assets from Dagster, use Pipes. Otherwise, use this."

File: docs/content/concepts/assets/external-assets.mdx (hunk 1 above)
Comment (jamiedemaria) [reply docs/content/concepts/assets/external-assets.mdx:126]: i can't think of anything better atm. "Dagster-managed"? "Dagster-materialized"? Listing out the actual decorators "@asset, @multi_asset, and @graph_asset with external ..."?

File: docs/content/concepts/assets/external-assets.mdx [hunk 2]
```diff
@@ -199,26 +221,50 @@ curl --request POST \
...
 
 response = requests.request("POST", url, json=payload, headers=headers)
 ```
 
+---
+
 </TabItem>
 </TabGroup>
 
-The API also has endpoints for reporting [asset observations](/concepts/assets/asset-observations) and [asset check evaluations](/concepts/assets/asset-checks).
```
Comment (jamiedemaria) [inline docs/content/concepts/assets/external-assets.mdx:220]: why remove?

**Review by jamiedemaria — APPROVED** [review]
these changes look good to me

File: docs/content/concepts/assets/external-assets.mdx (hunk 2 above)
Comment (erinkcochran87) [reply docs/content/concepts/assets/external-assets.mdx:220]: I just moved it up to come before the examples!

**Comment by yuhan** [comment]
- line 149: Should external_assets_from_specs have the asterisk?
  * yes
- line 313: Can report_runless_asset_event be used for non-external assets?
  * i believe technically yes, but not recommended which means it's doable but may pollute the non-external assets. disclaimer: i didn't have the time to actually test it - i just read the code of that method. so i can't 100% confirm it. so i think we can keep line 313 as-is.
- line 320: Any reason not to include the instance call (DagsterInstance.get())?
  * when it's inside, say, a sensor, instance can be access through context.instance rather than `DagsterInstance.get()`, so i think keeping it general as `instance.` is fine.

**Comment by schrockn** [comment]
> * i believe technically yes, but not recommended which means it's doable but may pollute the non-external assets. disclaimer: i didn't have the time to actually test it - i just read the code of that method. so i can't 100% confirm it. so i think we can keep line 313 as-is.

For sure. This is also how we will implement the new "report event" functionality in the UI.

**Pull request merged**